pkg-manager/hoist/src/index.ts

```diff
@@ -286,3 +330,12 @@ async function symlinkHoistedDependency (
   await symlinkDir(depLocation, dest)
   linkLogger.debug({ target: dest, link: depLocation })
 }
+
+function nameVerFromPath (depPath: string, publicHoistedModulesDir: string) {
+  // eslint-disable-next-line @typescript-eslint/no-require-imports, @typescript-eslint/no-var-requires
+  const manifest = require(path.join(path.dirname(publicHoistedModulesDir), depPath, 'package.json'))
```

Comment:
same, don't read the `package.json` file again, use the already read `package.json`.

Reply:
I can remove this one, is not used as the `depPath `constant is the name too.